test-requirements.txt

```diff
@@ -6,7 +6,6 @@ brotlipy==0.7.*
 cryptography
 flake8
 flake8-bugbear
-flake8-comprehensions
```

Comment:
Why drop this plugin?

Reply:
It's unneccessarily causing breakages...

Eg. expressions such as...

`self.content = b"".join([part async for part in self.stream()])`

Are failing with "C416 Unnecessary list comprehension - rewrite using list()."

Reply:
Can be resolved with `# noqa: C416`, as we want to keep the comprehension _there_, but perhaps not other places (non-async comprehensions).

Reply:
Related: https://github.com/adamchainz/flake8-comprehensions/issues/196

Reply:
Fix released if you want to add it back :)